src/SAML2/XML/saml/NameIDType.php

```diff
@@ -96,7 +95,7 @@ public function getFormat(): ?string
      * @param string|null $format
      * @return void
      */
-    public function setFormat(string $format = null): void
+    public function setFormat(?string $format): void
```

Comment:
Protected?

Also, is the format supposed to be an URI? Because in that case we should probably discard empty values at least.